go.mod

```diff
@@ -1,23 +1,62 @@
 module github.com/kubernetes-csi/csi-lib-utils
 
-go 1.16
+go 1.19
 
 require (
 	github.com/container-storage-interface/spec v1.5.0
 	github.com/golang/protobuf v1.5.2
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
-	google.golang.org/grpc v1.38.0
-	k8s.io/api v0.22.0
-	k8s.io/client-go v0.22.0
+	github.com/stretchr/testify v1.8.0
+	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10
```

Comment:
Can you change this to v0.4.0?  There's a CVE in earlier version.

Reply:
Sure let me do that

Reply:
You are going to update this to v0.4.0, right?

Reply:
This one has been constantly downgraded again. Let me update.